README.md

```diff
@@ -31,8 +31,8 @@ python3 -m pip install -r requirements.txt
 - Install virtualenv: `pip install virtualenv`.
 - Setup virtualenv:
```

Comment:
What I was thinking is that maybe somewhere after this line you could say, ‘Follow to steps below to setup virtualenv and in case you are facing errors view [these] steps to setup virtualenv and then add your steps.

Reply:
i already installed 'pip install virtualenv'


"""
Microsoft Windows [Version 6.1.7601]
Copyright (c) 2009 Microsoft Corporation.  All rights reserved.

C:\Users\User>pip install virtualenv
Requirement already satisfied: virtualenv in c:\users\user\appdata\local\program
s\python\python38\lib\site-packages (20.1.0)
Requirement already satisfied: six<2,>=1.9.0 in c:\users\user\appdata\local\prog
rams\python\python38\lib\site-packages (from virtualenv) (1.15.0)
Requirement already satisfied: appdirs<2,>=1.4.3 in c:\users\user\appdata\local\
programs\python\python38\lib\site-packages (from virtualenv) (1.4.4)
Requirement already satisfied: distlib<1,>=0.3.1 in c:\users\user\appdata\local\
programs\python\python38\lib\site-packages (from virtualenv) (0.3.1)
Requirement already satisfied: filelock<4,>=3.0.0 in c:\users\user\appdata\local
\programs\python\python38\lib\site-packages (from virtualenv) (3.0.12)
WARNING: You are using pip version 20.3.2; however, version 21.0.1 is available.

You should consider upgrading via the 'c:\users\user\appdata\local\programs\pyth
on\python38\python.exe -m pip install --upgrade pip' command.

C:\Users\User>
"""

after this i run      `virtualenv .venv -p python`
"""
D:\Lichess\lichessbot\lichess-bot-master>virtualenv .venv -p python
'virtualenv' is not recognized as an internal or external command,
operable program or batch file.

D:\Lichess\lichessbot\lichess-bot-master>
"""

Reply:
No, what my point over here is that with these commands you have provided my bot doesn’t work. With the previous commands it worked.

Reply:
The commands you provided only work in PowerShell for me (because ps1 is a PowerShell file). I know that lichess-bot suggests PowerShell but I use cmd. In cmd I use
```
virtualenv .venv -p python
.\.venv\Scripts\activate
```
I tried
```
virtualenv .venv -p python
./.venv/Scripts/activate
```
in PowerShell and it worked. You seem to be using cmd but I can't run ps1 files from cmd.

Reply:
./.venv/Scripts/Activate.ps1 (.\.venv\Scripts\activate.bat should work in cmd

Reply:
@TheYoBots 
Can you comment your cmd logs here with old commands

Reply:
No logs as such. Just the command ‘...’ doesn’t work. And the previous ones work better.

Reply:
Ok